docs/legacy_installation.md

```diff
@@ -0,0 +1,52 @@
+### Legacy installation (without SymfonyFlex)
```

Comment:
I wonder how we can make the installation instructions not being duplicated here.

I think we can make this process like:
 - follow the default installation instructions
 - do these steps manually: add bundles to the kernel, import config, import routing

Reply:
I was thinking about it as well, but then I thought that it would be nice not being forced to switch files to get the whole installation instructions (it could be especially burdensome for developers that use Sylius plugin the first time). That's why I've extracted the separate file, to not litter the main README, but also to have everything in one place.
IMO it's not a very big deal to maintain this separate file and it may be even better to have all process here if we go with some other installation automatizations (templates, migrations) in the future.